lib/routes/routeBackbeat.js

```diff
@@ -402,9 +402,26 @@ function putObject(request, response, log, callback) {
             if (contentMD5 !== md5) {
                 return callback(errors.BadDigest);
             }
+            // stealing some code from createAndStoreObject.js to
+            // create a valid locations array
+            const { key, dataStoreName, dataStoreType, dataStoreETag,
+                    dataStoreVersionId } = retrievalInfo;
+            const prefixedDataStoreETag = dataStoreETag
+                      ? `1:${dataStoreETag}`
```

Comment:
Just curious, why `dataStoreETag` has to be updated using `1:`  as a prefix?

Reply:
If I remember correctly, it is related to the put object copy part, to keep track of what is really a part locally (think nesting). I don't remember and I have to dig deeper to find out if it was only to support the NFS server's operation or to have the correct behavior with cloudserver as well